.github/workflows/promote-to-stable.yml

```diff
@@ -11,7 +11,7 @@ permissions:
 jobs:
   promote:
     name: ⬆️ Promote to stable
-    environment: "Candidate Branch"
+    environment: "LTS Branch"
     runs-on: ubuntu-latest
     if: |
       ( !github.event.issue.pull_request )
```

Comment:
This file can also be removed, you'll need to delete this on this branch, then update the promotion workflow on `candidate`